okapi-common/src/main/java/org/folio/okapi/common/ModuleId.java

```diff
@@ -88,6 +128,10 @@ public int hashCode() {
     return c;
   }
 
+  /**
+   * String representation of module ID
+   * @return string representation
+   */
```

Comment:
Please do not overwrite the inherited Javadoc (`@Override`) unless the new Javadoc is better.

Reply:
oh nice